File: src/test/java/com/futurice/testtoys/ExampleTest.java
package com.futurice.testtoys;

import static org.junit.Assert.assertTrue;

import java.io.IOException;

import org.junit.Test;

import com.futurice.testtoys.TestTool;

public class ExampleTest {	
	@Test
	public void testPlus() throws IOException  {
		TestTool t = new TestTool("testio/example/plus"); 

		t.tln("testing plus operation"); 
		t.tln("  1+1=" + (1+1));
  
        assertTrue(t.done());
	}
}

File: src/test/scala/com/futurice/testtoys/ExampleTest.scala
package com.futurice.testtoys

import org.junit.Assert.assertTrue
import java.io.IOException
import org.junit.Test

class ExampleTest extends TestSuite("example") {
  test("plus") { t =>
    t.tln("testing plus operation")
    t.tln("  1+1=" + (1 + 1))
  }
}